R/pkg/inst/tests/testthat/test_context.R

```diff
@@ -141,3 +141,8 @@ test_that("sparkJars sparkPackages as comma-separated strings", {
   expect_that(processSparkJars(f), not(gives_warning()))
   expect_match(processSparkJars(f), f)
 })
+
+test_that("sparkLapply should perform simple transforms", {
+  doubled <- spark.lapply(1:10, function(x){2 * x})
+  expect_equal(doubled, as.list(2 * 1:10))
+})
```

Comment:
Would be good to add a test where we capture some environment variables and/or use a package. Also we should update https://github.com/apache/spark/blob/master/docs/sparkr.md but we can open another JIRA for that I guess.